projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/boost-cache-actions.php

```diff
@@ -44,15 +45,15 @@
  */
 function jetpack_boost_delete_cache() {
 	$boost_cache = new Boost_Cache();
-	$boost_cache->invalidate_cache( Filesystem_Utils::DELETE_ALL );
+	$boost_cache->delete_recursive( home_url() );
 }
 
 /**
  * Delete cache for homepage and paged archives.
  */
 function jetpack_boost_delete_cache_for_home() {
 	$boost_cache = new Boost_Cache();
-	$boost_cache->invalidate_cache_for_front_page( Filesystem_Utils::DELETE_ALL );
+	$boost_cache->delete_page( home_url() );
```

Comment:
I was trying to make sense of the action and function name and decided it should only delete the homepage. But, the previous implementation was deleting the whole cache.

Reply:
This needs to replicate what the original `invalidate_cache_for_front_page` did. If the homepage shows a static page it needs to delete the "posts" page too. The stable version of the plugin deletes the "homepage" post too, but it might be enough to delete home_url() for that. I'll fix this.

Reply:
I just updated the PR to delete the posts page too.